### ISSUE TITLE
Checkboxes won't work!

### ISSUE THREAD

**Issue by sciclic** [STAT545-UBC/Discussion#544]
Hi,

I'm trying to add some checkboxes as part of the edition of the README file for Assignment 01, I've tried different things that I read online (e.g. using - [   ] instead of only [ ]) but I can't seem to make it work. Any ideas as to what's going on? 

Thank you!

**Comment by aleurcelay** [comment]
Hi @sciclic,
I am not sure why are your checkboxes not rendering but you could try this instead:
`* [x]  checked`
`* [ ]  unchecked`

* [x]  checked
* [ ]  unchecked

**Comment by sciclic** [comment]
Thanks for your reply @aleurcelay. I did try that but still not working :cry:, I'll just do something else instead and I'll go to the TA hour after the next lecture to try and figure it out.

**Comment by vincenzocoia** [comment]
Yeah, checkboxes in markdown don't actually allow you to actively check or uncheck the box. This is because this would require changing the markdown source code from `[ ]` to `[x]` (or vice versa). It's different with GitHub Issues, though -- they've implemented a convenience, where clicking on the box actually edits the comment, and further, tracks that you made the edit. Try clicking on the checkboxes in @aleurcelay's post above to see what I mean!
